src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -69,23 +69,29 @@ void WaveformRenderMark::draw(QPainter* painter, QPaintEvent* /*event*/) {
 
                 // Check if the current point need to be displayed
```

Comment:
```suggestion
                // Check if the current point needs to be displayed
```
Please fix this typo in the existing comment